scripts/build

```diff
@@ -0,0 +1,85 @@
+#!/usr/bin/env bash
+# execute all of the steps necessary to build the project
+set -e #-x
+
+usage() {
+  cat <<-'EOF'
+usage: build [-t|--skip-tests] [-h|--help] BUILDTYPE
+
+[-t|--skip-tests]       skip running unit tests;
+                        this is used in the CI process
+[-h|--help]             print this help text
+
+BUILDTYPE is one of the following:
+- project               the default build command
+- docker                build a docker image
+EOF
+}
+
+# script variables and defaults
+source $(dirname $0)/init.sh
+skip_tests=""
```

Comment:
We don't need skip_tests anymore in build because it's now part of `run`.

Reply:
this is needed as a part of the `build_docker()` and `build_project()` functions to skip tests while building